vault/example/example.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"os"
+
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/ONSdigital/go-ns/vault"
+)
+
+func main() {
+
+	log.Namespace = "vault-example"
+	log.Debug("Creating vault client", log.Data{})
+	devAddress := "http://127.0.0.1:8200"
```

Comment:
this should be `vaultAddress := os.Getenv("VAULT_ADDR")`

Reply:
done